core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolverTest.java

```diff
@@ -238,6 +247,36 @@ private static <T> void testMethod(Method method, T bean) {
         }
     }
 
+    @SuppressWarnings("unchecked")
+    private static void testResolver(AnnotatedValueResolver resolver,
+                                     Map<String, AttributeKey<?>> successExpectAttrKeys,
+                                     Map<String, AttributeKey<?>> failExpectAttrKeys) {
```

Comment:
Can we just merge `testResolver(AnnotatedValueResolver resolver)` into this method? I don't see a reason to chain calls from this method to another.

Reply:
I see, i used `@BeforeEach` annotation to inject test values to member fields `Map<String, AttributeKey<?>> successExpectAttrKeys`, `Map<String, AttributeKey<?>> failExpectAttrKeys` instead of `testresolver(AnnotatedValueResolver resolver, Map<String, AttributeKey<?>> successExpectAttrKeys,Map<String, AttributeKey<?>> failExpectAttrKeys))`. 

However, it could not be fully integrated with case of `@Param`, `@Header`. because i added test code which expect to throw error case when `@Attribute` are tested. 

If case which expect to throw error on`@Attribute`, is not suitable, i'm willing to remove it. i,e, `success` case only. Please let me know. 🙇‍♂️